### PR TITLE
IDEX-4121: Add BLANK project type handler will be created README file…

### PR DESCRIPTION
**Pull request by vparfonov**
… inside it

### CODE REVIEW

**Comment by vparfonov** [comment]
@skabashnyuk

File: platform-api/che-core-api-project/src/test/java/org/eclipse/che/api/project/server/handlers/CreateBaseProjectTypeHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+/**
+ *  @author Vitalii Parfonov
+ */
+@Listeners(value = {MockitoTestNGListener.class})
+public class CreateBaseProjectTypeHandlerTest {
```
Comment (skabashnyuk) [inline platform-api/che-core-api-project/src/test/java/org/eclipse/che/api/project/server/handlers/CreateBaseProjectTypeHandlerTest.java]: You don't need MockitoTestNGListener if you making mocks explicit.

File: platform-api/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/handlers/CreateBaseProjectTypeHandler.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    @Override
+    public String getProjectType() {
+        return Constants.BLANK_ID;
+    }
+
+    private byte[] getReadmeContent() {
+        String filename = "README.blank";
+        try {
+            return Resources.toByteArray(getClass().getClassLoader().getResource(filename));
```
Comment (skabashnyuk) [inline platform-api/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/handlers/CreateBaseProjectTypeHandler.java]: Resources.toByteArray(Resources.getResource("README.blank"));
wdyt?

Comment (vparfonov) [inline platform-api/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/handlers/CreateBaseProjectTypeHandler.java]: +1

File: platform-api/che-core-api-project/src/test/java/org/eclipse/che/api/project/server/handlers/CreateBaseProjectTypeHandlerTest.java (hunk 1 above)
Comment (vparfonov) [inline platform-api/che-core-api-project/src/test/java/org/eclipse/che/api/project/server/handlers/CreateBaseProjectTypeHandlerTest.java]: ok

**Pull request merged**